lib/modules/manager/helmfile/extract.spec.ts

```diff
@@ -355,9 +356,10 @@ describe('modules/manager/helmfile/extract', () => {
           },
           {
             currentValue: '0.4.2',
-            depName: 'url-example',
+            depName: 'ghcr.io/example/oci-repo/url-example',
```

Comment:
Is this change really necessary? The full name is already in `packageName`

Reply:
In the case of a package name replacement it is needed as the `auto-replace` uses the `depName` for the replacement (which is correct as in the case of the indirection via the repository, only the `depName` can be replaced).